src/status_im/ui/screens/browser/default_dapps.cljs

```diff
@@ -78,7 +78,11 @@
              :photo-path  "contacts://superrare"
              :description "Buy, sell and collect unique digital creations by artists around the world"}]}
    {:title (i18n/label :t/default-dapps-fun-games)
-    :data  [{:name        "CryptoKitties"
+    :data  [{:name        "Dragonereum"
```

Comment:
It breaks up the CK listing, can it be moved below `Crypto Takeovers`? (we also need to re-alphabetize the lists)

Reply:
Done